### ISSUE TITLE
"any" mappings aren't loaded from file

### ISSUE THREAD

**Issue by mnbf9rca** [the-via/keyboards#2236]
Probably i'm not doing this correctly - i looked to try and find some documentation on the keyboard layout definition file, but i can't find it.

I have a mac UK keyboard (a Keychron Q6 Pro), and I want to map layer 1 F3 and F4 to `0xc1` and `0xc2` respectively. To do this, i've edited the mapping file: [correct_mappings.json](https://github.com/the-via/keyboards/files/14951131/correct_mappings.json). But when i load this file, the mappings for for L1 F3/F4 dont load:
<img width="398" alt="image" src="https://github.com/the-via/keyboards/assets/1039058/922814c6-0697-44e7-b17d-97ff52f3b794">

I'm using the [keyboard definition file from the Keychron repo](https://github.com/Keychron/qmk_firmware/blob/bluetooth_playground/keyboards/keychron/q6_pro/via_json/q6_pro_iso_encoder.json).

**Comment by Xelus22** [comment]
use the ANY key under the SPECIAL tab.

**Comment by mnbf9rca** [comment]
Yes I get that. I’ve set the keys and then hit “save” to download the json, but when I reload it, the mapping is not there.

**Comment by XelorR** [comment]
+1

The same issue with Caps Word (0x7C73) button.
Looks like hex values can be exported to json file but disappears on loading.
